acceptable/tests/test_openapi.py

```diff
@@ -66,9 +61,87 @@ def test_convert_str_returns_same_value():
         assert "beeblebrox" == result
 
 
+class TidyStringTests(testtools.TestCase):
```

Comment:
Maybe add one extra test for `tidy_string()` with newlines?

Reply:
Done.